exercises/secret-handshake/description.md

```diff
@@ -23,7 +23,4 @@ Here's a couple of examples:
 Given the input 3, the function would return the array
 ["wink", "double blink"] because 3 is 11 in binary.
 
-Given the input 19, the function would return the array
-["double blink", "wink"] because 19 is 10011 in binary.
-Notice that the addition of 16 (10000 in binary)
-has caused the array to be reversed.
+The addition of 16 (10000 in binary) would reverse the array. So given the input 19, the function would return the array ["double blink", "wink"] because 19 is 10011 in binary.
```

Comment:
16 and 19 are mentioned, yes. I'd only reverse the order in which they are mentioned.

For someone familiar with binary, mentioning the "3" part might possibly seem unnecessary. But for someone unfamiliar with binary it might not be immediately obvious that 19 can be viewed as 16+2+1. Jumping from the example of 3 to the example of 19 might first seem unconnected and random. Without that additional information in the back of one's mind, some beginner might not immediately see that for the result for 19 one simply needs to reverse the result for 3 which is already provided and would instead begin their computation from scratch again. After a reminder that 19=3+16 and that the addition of 16 delivers the reverse, it becomes more apparent that the example of 19 wasn't chosen arbitrarily. You're right in that this additional information is not strictly necessary, since it's implied by what's already been said, but I think mentioning it nonetheless removes a likely source of confusion.

Reply:
I think this is a good change. I mean to say that even further explanation could be made.

If we look at the change here:

```
The addition of 16 (10000 in binary) would reverse the array. So given the input 19, the function would return the array ["double blink", "wink"] because 19 is 10011 in binary.
```
It mentions 19 and 16, but does not additionally explain the logical link from 19 to 16.  It could be explicit, as you've been in the comment above, about 19 being seen as 16+2+1 in binary.

For example only:

```
So given the input 19 is 10011 in binary, we can understand this as being equivalent to 16 + 2 + 1. Thus, recalling that 16 reverses the array, the function would return the array ["double blink", "wink"], ( 16 (10000 in binary).
```

Reply:
@davidculley What do you think about the above suggestion?

Reply:
I like your suggestions 👍 I'm glad that my proposed change is a good change.

@wolf99 Sure, we can make it explicit that `10011` can be seen as 16+3 or 16+2+1. It doesn't hurt to be more explicit. Perhaps I misunderstood you in my previous answer. Your suggestion looks good to me.

I was on winter vacation, but now I'm back. Thanks for your patience.